Setup.sh

```diff
@@ -59,12 +61,12 @@ if [ -d "${SPATIAL_DIR}" ]; then
 fi
 
 echo "Retrieve dependencies"
-spatial package retrieve tools       schema_compiler-x86_64-macos        "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/tools/schema_compiler-x86_64-macos.zip
-spatial package retrieve schema      standard_library                    "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/schema/standard_library.zip
-spatial package retrieve worker_sdk  c_headers                           "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c_headers.zip
-spatial package retrieve worker_sdk  c-dynamic-x86_64-clang-macos        "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c-dynamic-x86_64-clang-macos.zip
-spatial package retrieve worker_sdk  c-static-fullylinked-arm-clang-ios  "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c-static-fullylinked-arm-clang-ios.zip
-spatial package retrieve worker_sdk  csharp                              "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/csharp.zip
+spatial package retrieve tools       schema_compiler-x86_64-macos        "${PINNED_CORE_SDK_VERSION}"  ${DOMAIN_ENVIRONMENT_VAR}  "${CORE_SDK_DIR}"/tools/schema_compiler-x86_64-macos.zip
```

Comment:
@jessicafalk I haven't been able to test this, does this change look alright to you?